.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+        exclude: (migrations/|fixtures/)
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: no-commit-to-branch
+        args: ["--branch", "prod", "--branch", "dev"]
+
+-   repo: https://github.com/PyCQA/flake8
+    rev: 7.0.0
+    hooks:
+    -   id: flake8
+        args: ["--ignore=E203,E501,W503"]
+        exclude: (migrations/|fixtures/)
```

Comment:
the migrations scripts are still just python, I think it's worth formatting them too (especially since we occasionally need to write or edit them by hand)